File: 1360-number-of-days-between-two-dates/1360-number-of-days-between-two-dates.py
import datetime
class Solution:
    def daysBetweenDates(self, date1: str, date2: str) -> int:
        date1 = datetime.datetime.strptime(date1, '%Y-%m-%d').date()
        date2 = datetime.datetime.strptime(date2, '%Y-%m-%d').date()
        return abs((date1-date2).days)

File: 1360-number-of-days-between-two-dates/1360-number-of-days-between-two-dates.js
/**
 * @param {string} date1
 * @param {string} date2
 * @return {number}
 */
var daysBetweenDates = function(date1, date2) {
    date1=new Date(date1)
    date2=new Date(date2)
    diffTime=Math.abs(date2-date1)
    console.log(diffTime)
    diffDay=Math.ceil(diffTime/(1000*60*60*24))
    return diffDay
};